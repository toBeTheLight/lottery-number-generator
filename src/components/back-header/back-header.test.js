import React from 'react';
import ReactDom from 'react-dom';
import BackHeader from './back-header';
import { mount } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import { HashRouter as Router} from 'react-router-dom'

const div = document.createElement('div');

describe('back-header.js', () => {
  it('renders without crashing', () => {
    const wrapper = mount(
      <Router>
        <BackHeader name="页面标题"/>
      </Router>
    )
    expect(renderToJson(wrapper)).toMatchSnapshot();
    const h = wrapper.find('.am-navbar-title');
    expect(h.text()).toBe('页面标题');
  });
})