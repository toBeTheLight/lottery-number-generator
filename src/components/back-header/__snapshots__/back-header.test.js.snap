// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`back-header.js renders without crashing 1`] = `null`;
